src/lib.rs

```diff
@@ -1,37 +1,37 @@
-#[deny(missing_docs)]
-
-/**
-* `lychee` is a library for checking links.
-* "Hello world" example:
-* ```
-* use std::error::Error;
-*
-* #[tokio::main]
-* async fn main() -> Result<(), Box<dyn Error>> {
-*   let response = lychee::check("https://github.com/lycheeverse/lychee").await?;
-*   println!("{}", response);
-*   Ok(())
-* }
-* ```
-*
-* For more specific use-cases you can build a lychee client yourself,
-* using the `ClientBuilder` which can be used to
-* configure and run your own link checker and grants full flexibility:
-*
-* ```
-* use lychee::{ClientBuilder, Status};
-* use std::error::Error;
-*
-* #[tokio::main]
-* async fn main() -> Result<(), Box<dyn Error>> {
-*   let client = ClientBuilder::default().build()?;
-*   let response = client.check("https://github.com/lycheeverse/lychee").await?;
-*   assert!(matches!(response.status, Status::Ok(_)));
-*   Ok(())
-* }
-* ```
-*/
+//!
```

Comment:
Great to make this a doc-comment!

But isn't a blank line here creating a newline in the rendered docs? Just curious.

Reply:
Didn't check yet, but I assumed cargo would be smart enough to trim empty lines at the beginning and the end.